### DISCUSSION TITLE
Combined orientation in one pdf

### DISCUSSION THREAD

**Discussion by svitlanaburlak** [dompdf/dompdf#3248] [Q&A] [answered]
Hello, 
I need to have all my pages in landscape orienation but one, in the middle of the document, in portrait. 
Is it possible to do it with just dompdf functionality ?
Or the solution is to use pdf merger ?

Thanks.

**Comment by bsweeney** [comment, marked as answer]
Currently you would need to generate the different orientations separately. We do have a pending update to the PDF generation logic that would allow this (#613) and I am investigating an option that should make this possible (#498) but do not yet have a time frame for development.
